broker/cloud_functions/ps_to_gcs/requirements.txt

```diff
@@ -3,7 +3,29 @@
 # dependencies for a Cloud Function must be specified in a `requirements.txt`
 # file (or packaged with the function) in the same directory as `main.py`
 
-fastavro
+fastavro==1.4.4  # fastavro==1.4.4 is known to work for ztf (see below)
```

Comment:
Is this really necessary for ELAsTiCC? We want to avoid pinning to old versions like this if at all possible.

Reply:
I don't believe it is, but it is defined that way in the `develop` branch. I was trying to keep things consistent between branches

Reply:
Ok, let's leave it unpinned here and deal with the conflict if/when we merge the two branches.